integration-tests/tx-rollup-l2-address.spec.ts

```diff
@@ -69,7 +69,7 @@ CONFIGS().forEach(({lib, setup, protocol}) => {
       done();
     })
 
-    jakartanet(`jakarta contract with params and storage as tx_rollup_l2_address`, async (done) => {
+    limanet(`jakarta contract with params and storage as tx_rollup_l2_address`, async (done) => {
```

Comment:
Should this test run on all testnets? Using `it` rather than `limanet`?
I am not too sure why they were not running on Kathmandu..